CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
 # 4.2
 * [Improve Parlot error handling](https://github.com/ncalc/ncalc/pull/181) by [Andrey Bykiev](https://github.com/Bykiev)
+* [Fix OverflowException with double values](https://github.com/ncalc/ncalc/pull/188) by [Andrey Bykiev](https://github.com/Bykiev), [Gustavo Mauricio de Barros](https://github.com/gumbarros)
+* [Fix double value precision loss](https://github.com/ncalc/ncalc/pull/188) by [Andrey Bykiev](https://github.com/Bykiev), [Gustavo Mauricio de Barros](https://github.com/gumbarros)
+* [Add new ExpressionOptions.AllowBooleanCalculation option. ExpressionOptions.CaseInsensitiveComparer is now obsolete, please use ExpressionOptions.OrdinalStringComparer instead](https://github.com/ncalc/ncalc/pull/188) by [Gustavo Mauricio de Barros](https://github.com/gumbarros)
```

Comment:
It's not obsolete and non breaking, I just added CaseInsensitiveStringComparer to match the naming with OrdinalStringComparer. CaseInsensitiveComparer still exists but with obsolete attribute

Reply:
Sorry, don't understand which change about you are not agreed

Reply:
The breaking change section is below

Reply:
OrdinalStringComparer is not related to CaseInsensitiveComparer, is a totally new feature

Reply:
Thanks, should be CaseInsensitiveStringComparer, but it's not in a breaking change section